spec/form_models/jobseekers/job_application/employment_history_form_spec.rb

```diff
@@ -1,5 +1,67 @@
 require "rails_helper"
 
 RSpec.describe Jobseekers::JobApplication::EmploymentHistoryForm, type: :model do
-  it { is_expected.to validate_presence_of(:employment_history_section_completed) }
+  subject(:form) { described_class.new(attributes) }
+
+  let(:attributes) do
+    {
+      employment_history_section_completed: employment_history_section_completed,
+      unexplained_employment_gaps_present: unexplained_employment_gaps_present,
+      unexplained_employment_gaps: unexplained_employment_gaps,
+    }
+  end
+
+  let(:employment_history_section_completed) { "true" }
+  let(:unexplained_employment_gaps_present) { "true" }
+  let(:unexplained_employment_gaps) do
+    {
+      gap1: { started_on: Date.new(2023, 12, 1), ended_on: Date.new(2024, 12, 1) },
+      gap2: { started_on: Date.new(2020, 11, 1), ended_on: Date.new(2021, 5, 31) },
+    }
+  end
+
+  describe "validations" do
+    it { is_expected.to validate_presence_of(:employment_history_section_completed) }
+
+    context "when employment history gaps are present" do
+      it "adds errors for each unexplained gap" do
+        form.valid?
```

Comment:
the idiom here to to assert that the form isn't valid, which then allows the errors to be asserted afterwards rather than this hanging valid? call